entproto/internal/entprototest/fieldmapper_test.go

```diff
@@ -41,6 +41,8 @@ func (suite *AdapterTestSuite) TestFieldMap() {
 	assert.EqualValues("BlogPosts", blogPosts.PbStructField())
 	assert.False(blogPosts.IsIDField)
 	assert.True(blogPosts.IsEdgeField)
+	assert.EqualValues("Id", blogPosts.EdgeIDPbStructField())
```

Comment:
Is it common to write the identifier field this way, and not `ID`?

Reply:
Yes this ist he way protoc generated Go code from a field named `id`